code/game/objects/items/stacks/spiketrap.dm

```diff
@@ -0,0 +1,156 @@
+/obj/item/stack/spiketrap //A deployable spike trap, causes damage to anyone who steps over it
+	name = "spike trap assembly"
+	desc = "An assortment of sharp, metal spikes that stabs any unfortunate sole (or soul) that steps on it."
+	icon = 'icons/Marine/traps.dmi'
+	icon_state = "spiketrap"
+	flags_item = NOBLUDGEON
+	singular_name = "pile"
+	w_class = WEIGHT_CLASS_SMALL
+	force = 0
+	attack_verb = list("hit", "whacked", "sliced")
+	max_amount = 20
+	merge_type = /obj/item/stack/spiketrap
+	max_integrity = 200
```

Comment:
The integrity is 2x razorwire, when it's supposed to be a more offensive alternative, which also can't be easily deleted with acid spray, and is even more spammable thanks to it's low costs.